bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -36,14 +50,18 @@
       <div class="content-wrapper">
         <div class="phone">
           <div class="fxos-cta">
-            <h1><span>{{_('Firefox OS')}}</span></h1>
+            <h1>{{ high_res_img('img/firefox/os-new/fxos-wordmark.png', {'alt': _('Firefox OS'), 'width': '420', 'height': '85'}) }}</h1>
```

Comment:
I think in this case an SVG may actually prove smaller. I did consider it, but given that it is the page `<h1>` I decided to play it safe in terms of format for older browsers. 

I **could** be persuaded to not care about those dusty old browsers though...